### PR TITLE
EN-1913-digital-assets-update-without-file-delete-saved-metadata

### PR DESCRIPTION
**Pull request by alepintus**
Hey @joewhite101 @eugeniosant ,
Saving metadata when updating an existing resource fixed.

Could you please have a look?
Thanks!

### CODE REVIEW

File: plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/resource/MultipleResourceAction.java [hunk 1]
```diff
@@ -103,6 +103,7 @@ public void validate() {
 
     @Override
     public String edit() {
+        logger.debug("Edit");
```
Comment (joewhite101) [inline plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/resource/MultipleResourceAction.java]: This probably needs some context. "Edit in multiple resource action for id {} ...." or something like that

File: plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/resource/MultipleResourceAction.java [hunk 2]
```diff
@@ -148,15 +149,30 @@ protected void validateFileDescriptions() {
     }
 
     public String joinCategory() {
+        logger.debug("joinCategory");
```
Comment (joewhite101) [inline plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/resource/MultipleResourceAction.java]: Add some context on location and id here.

File: plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/resource/MultipleResourceAction.java [hunk 3]
```diff
@@ -148,15 +149,30 @@ protected void validateFileDescriptions() {
...
         getFileDescriptions();
+        fetchMetadataEdit();
         return super.joinCategory();
     }
 
     public String removeCategory() {
+        logger.debug("removeCategory");
```
Comment (joewhite101) [inline plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/resource/MultipleResourceAction.java]: Add context to log message

File: plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/resource/MultipleResourceAction.java [hunk 4]
```diff
@@ -194,6 +210,7 @@ protected boolean isValidType(String docType, String[] rightTypes) {
 
     @Override
     public String save() {
+        logger.debug("Save");
```
Comment (joewhite101) [inline plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/resource/MultipleResourceAction.java]: Add context

**Comment by entando-service** [comment]
All is Well

**Comment by alepintus** [comment]
Hey @joewhite101 ,
Thanks for the review, logs changed as suggested.

**Comment by entando-service** [comment]
All is Well

**Pull request merged**